clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -17297,6 +17297,14 @@ Value *CodeGenFunction::EmitPPCBuiltinExpr(unsigned BuiltinID,
     Value *Op1 = EmitScalarExpr(E->getArg(1));
     Value *Op2 = EmitScalarExpr(E->getArg(2));
     Value *Op3 = EmitScalarExpr(E->getArg(3));
+    if (BuiltinID == PPC::BI__builtin_ppc_rldimi &&
```

Comment:
nit: please add comments here why we don't generate `Intrinsic::ppc_rldimi` for PPC32.